Building the conda recipe does not work on Windows

From the bokeh git repo, running `conda build conda.recipe` won't work on Mac, Windows, or Linux as there is no build of `pdiff`.

Running into this build problem on Mac.

    npm WARN package.json Dependency 'coffee-script' exists in both dependencies and devDependencies, using 'coffee-script@^1.9.3' from dependencies
    npm WARN package.json Dependency 'underscore' exists in both dependencies and devDependencies, using 'underscore@^1.5.2' from dependencies
    npm ERR! Darwin 13.4.0
    npm ERR! argv "/zopt/conda/envs/_build/bin/node" "/zopt/conda/envs/_build/bin/npm" "install"
    npm ERR! node v4.1.0
    npm ERR! npm  v2.14.3
    npm ERR! code EPEERINVALID
    
    npm ERR! peerinvalid The package browserify@11.2.0 does not satisfy its siblings' peerDependencies requirements!
    npm ERR! peerinvalid Peer browserify-shim@3.8.5 wants browserify@>= 2.3.0 < 10
    
    npm ERR! Please include the following file with any support request:
    npm ERR!     /Users/kirkhamj/Developer/PyCharmCE/bokeh/bokehjs/npm-debug.log

In regards to the
```
npm ERR! peerinvalid The package browserify@11.2.0 does not satisfy its siblings' peerDependencies requirements!
npm ERR! peerinvalid Peer browserify-shim@3.8.5 wants browserify@>= 2.3.0 < 10
```
error, I was able to fix it on Mac by deleting an old copy of the `browserify-shim/` package out of my `node_modules/` directory

--- a/conda.recipe/bld.bat
+++ b/conda.recipe/bld.bat
@@ -1,6 +1,11 @@
 set BLD_DIR=%cd%
 cd /d %RECIPE_DIR%\..
 
+cd bokehjs
+npm install
+if errorlevel 1 exit 1
+cd ..
+
 python scripts/get_bump_version.py > __conda_version__.txt
 copy /Y __conda_version__.txt %BLD_DIR%
 if errorlevel 1 exit 1

--- a/conda.recipe/meta.yaml
+++ b/conda.recipe/meta.yaml
@@ -18,7 +18,7 @@ requirements:
     - python
     - distribute
     - setuptools
-    - nodejs >=4.1  [unix]
+    - nodejs >=4.1
     - six
     - python-dateutil
     - numpy

